pkg/kubelet/container/runtime.go

```diff
@@ -84,6 +84,8 @@ type Runtime interface {
 	ContainerCommandRunner
 	// ContainerAttach encapsulates the attaching to containers for testability
 	ContainerAttacher
+	// Get Container NetNamespacePath
+	GetNetNs(containerId string) (string, error)
```

Comment:
Given that this REALLY only applies to the Docker runtime, does it belong in the Runtime interface?  The eventual destination is that the CNI logic lives entirely withing the docker runtime, right?

What if we changed the callers that need this to `plugin.host.GetRuntime().(*DockerRuntime).GetNetNs(string(id))`?  It's a little horrible during the transition, but it means we don't have to pollute, for example, the nascent Hyper runtime.

@dchen1107 @yujuhong since they are deep into Runtime stuff.  Happy to give you context f2f tomorrow...